packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -77,7 +79,11 @@ export interface SdkOperationGroup {
   service: Namespace;
 }
 
-interface SdkTypeBase {
+interface DecoratedType {
+  decorators: Record<string, Array<any>>;
+}
+
+interface SdkTypeBase extends DecoratedType {
```

Comment:
is inheritance a good practice here? could this be another property?

Reply:
same with previous reply.